web/resources/views/next/base.blade.php

```diff
@@ -0,0 +1,172 @@
+{{-- -
+    All arguments are optional.
+
+    data:
+        $social_url
+        $social_title
+        $social_description
+        $social_image
+        $social_type
+        $social_twitter_card
+        $social_twitter_site
+        $social_twitter_creator
+        $social_rating
+
+        $title
+        $robots
+        $canonical
+        $license
+
+        $theme_color
+
+        $favicon_svg
+        $favicon_png
+        $favicon_apple
+        $favicon_mask
+
+    sections:
+        content
+
+    stacks:
+        preloads
+        styles
+        head
+        scripts
+- --}}
+
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+
+<head>
+    {{-- Functionality --}}
+    <meta charset="utf-8">
+    <base href="/">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    {{-- Security --}}
+    <meta name="referrer" content="no-referrer">
+    <meta name="csrf-token" content="{{ csrf_token() }}" />
+    {{-- TODO:
+    <meta
+        http-equiv="Content-Security-Policy"
+        content=""
+    > --}}
+
+    {{-- Preloads, Preconnects --}}
+    {{-- TODO: figure out how to properly preload these:
+        <link rel="preload" href="{{ vite_asset('resources/css/base.scss') }}" as="style">
+        <link rel="preload" href="{{ vite_asset('resources/scripts/index.ts') }}"
+              as="script">
+        <link rel="preload"
+            href="{{ vite_asset('resources/fonts/variable/PublicSans-VariableFont.woff2') }}"
+            as="font" type="font/woff2" crossorigin>
+        <link rel="preload"
+            href="{{ vite_asset('resources/fonts/variable/Exo2-VariableFont.woff2') }}"
+            as="font" type="font/woff2" crossorigin>
+    - --}}
+    {{-- TODO: preload the user's locale file --}}
+    @stack('preloads')
+
+    {{-- OpenGraph --}}
+    {{-- TODO: verify that getLocale returns a well formed string for this --}}
+    <meta property="og:locale" content="{{ app()->getLocale() }}">
+    @isset($social_url)
+        <meta property="og:url" content="{{ $social_url }}">
+    @endisset
+    @isset($social_title)
+        <meta property="og:title" content="{{ $social_title }}">
+    @endisset
+    @isset($social_description)
+        <meta name="description" content="{{ $social_description }}">
+        <meta property="og:description" content="{{ $social_description }}">
```

Comment:
Are these two always the same? From what I understand they can be different in some circumstances. (Nitpick)

Reply:
I'm not sure _why_ they would be different. The only reason to make them different is if you want Facebook/Twitter/Discord/etc. descriptions to be different from like, Google and browser bookmark descriptions.

I mean, maybe you do, but the fact that different services pull from either `description` or `og:description` (and you won't know which), means that the only sane thing to do (IMO) is to set them both to the same value.